.github/workflows/assets.yml

```diff
@@ -58,6 +58,8 @@ jobs:
              EVE=lfedge/eve:${TAG}-${HV}-${{ env.ARCH }}
              docker pull "$EVE"
           else
+             sudo apt-get update
+             sudo apt-get install golang
```

Comment:
Instead of this, can we just use the [go setup action](https://github.com/actions/setup-go)?

Reply:
What is the advantage of it?

Reply:
Consistent with the way of doing things in Actions.

Reply:
I see -- https://github.com/lf-edge/eve/pull/4076#pullrequestreview-2168901207

Just a small comparison of the advantages:


-| Github Action  | Run `apt-get` directly
---| -------------            | ------------- 
Lines| 968 unknown lines           | < 10 lines  
Well-known language in the project| written in shell | written in typescript  
Consistent with | doing things in Actions | installing software as we already do
Dependency | tightly coupled to Github | easy migration to new platform

(https://github.com/actions/setup-go/blob/main/src/system.ts#L33 - does this work for arm64?)

Reply:
It has to, there are thousands (millions) of daily arm64 runs, so it must work. I have used it a lot.

> easy migration to new platform

Heh, not a chance that it is easy. Look at all of those workflow files. This pipeline is tightly integrated into Actions far beyond this.

Reply:
What it does do, is give precise control over the version, which is why it is so popular.

Reply:
Okay, let's see if this works ...

Reply:
Seems, it does not work https://github.com/lf-edge/eve/actions/runs/9878843563/job/27283688482?pr=4063#step:9:10

(although on amd64 and riscv64 it works).